cmd/analyze/main.go

```diff
@@ -244,4 +255,17 @@ func main() {
 		slog.InfoContext(ctx, "Starting dynamic analysis")
 		dynamicAnalysis(ctx, pkg, &resultStores)
 	}
+
+	return nil
+}
+
+func main() {
+	if err := run(); err != nil {
+		if errors.As(err, &usageError{}) {
+			fmt.Fprintf(os.Stderr, "Usage error: %v\n", err)
```

Comment:
Might as well os.Exit(2) here, since you can

Reply:
haha okay